Make all vals in Ops classes private

jhnsmth opened the pull request.

benhutchison commented:
See conversation at https://github.com/typelevel/mouse/pull/63 re `private`

The build is broken because we're on Scalajs 0.6.31 which hasnt been published for Scala 2.12.11.

I think mouse should migrate to Scalajs 1.x, which then enables upgrade to 2.12.11

jhnsmth commented:
Sorry I did not see that PR, but I believe that it's worth doing because this vals pollute auto-complete scope in IDE. So when you type this useless vals always pop-up, stealing space from some useful methods. This is the problem I wanted to address.

If you do not agree, close this PR. And I will open a new one with scalajs update

benhutchison commented:
Actually, thats a good justification; all I wanted from the previous PR was a clear reason to add the additional code change, and you've given me one 👍

jhnsmth commented:
I've removed commit with scala verison bump to make CI pass and split changes. I'll send it in another PR along with other necessary changes to get scalajs 1.x

benhutchison commented:
Thank you!
